packages/utils/interactive-hooks/src/useKeyProps.types.macos.ts

```diff
@@ -0,0 +1,33 @@
+import { NativeSyntheticEvent } from 'react-native';
+
+// React Native macOS doesn't have typescript types yet, so define the type here.
+interface NativeKeyEvent {
+  // Modifier keys
+  capsLockKey: boolean;
+  shiftKey: boolean;
+  ctrlKey: boolean;
+  altKey: boolean;
+  metaKey: boolean;
+  numericPadKey: boolean;
+  helpKey: boolean;
+  functionKey: boolean;
+  // Key options
+  ArrowLeft: boolean;
+  ArrowRight: boolean;
+  ArrowUp: boolean;
+  ArrowDown: boolean;
+  key: string;
+}
+
+export type KeyPressEvent = NativeSyntheticEvent<NativeKeyEvent>;
+
+export type KeyCallback = (e?: KeyPressEvent) => void;
+
+export type KeyPressProps = {
```

Comment:
Why do we have Windows type definitions in a macOS-specific file?

Reply:
Not windows. I converted the macOS flow type to typescript and put it here. Post 0.71, I hope to import the type directly from RN-macOS